test/testdata/lsp/prop_references.rb

```diff
@@ -45,6 +45,6 @@ def example
 b.some_const
 # ^^^^^^^^^^ usage: B#some_const
 b.some_prop=('')
-# ^^^^^^^^^^ usage: B#some_prop
+# ^^^^^^^^^  usage: B#some_prop
```

Comment:
Kind of curious whether this parses directly as the call to `some_prop=`.  I assume since we have overlapping locs for getters and setters that it's hard to disambiguate here.